wordpress-mobile/WordPress-Android
# [Jetpack Plugin Full Install] Remove Jetpack Full Plugin Install feature flag

To test:
The feature should continue to be working as expected, so basically the `Test steps` section in the feature test plan pctCYC-Gx-p2

See the `Prerequisites` section in that same link to understand how to create a self-hosted site, deactivate the Jetpack full plugin and install an individual Jetpack plugin.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and updated unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated `GetShowJetpackFullPluginInstallOnboardingUseCaseTest.kt`

PR submission checklist:


UI Changes testing checklist:
